EIPS/eip-5503.md

```diff
@@ -0,0 +1,176 @@
+---
+eip: 5503
+title: Refundable Token Standard
+author: <admin@startfund.io>
+status: Draft
+type: Standards Track
+category: ERC
+created: 2022-08-16
+---
+
+## Abstract
+The value of a token can be the total sum of the linked currency’s value. For example, in the Token issuing process, the issuer can receive money from buyers( or investors) and transfer issuing token to buyers. . If the offering process is completed, there is no issue. But buyers can change their plan, or the offering is failed(or be canceled) cause of misfitting the compliance rules or other rules. There is no way guarantee to pay back (refund) to the buyer in the on-chain network.
+We have suggested this process make possible in on-chain network with a payable currency like token(ex: USDT)
+
+## Motivation
+A standard interface allows the payable token contract to interact with ERC-2000 interface within smart contracts.
```

Comment:
[editorial] This on the other hand seems  a better fit of abstract section
[editorial] also, please update the ERC-2000 to a placeholder or real ERC number. 
[editorial] currently editors mandate the ERCs to be specified in the format of "EIP-XXX" plus the link to the file.

Reply:
updated for ERC-XXX rules and simplify `Motivation` part.